### ISSUE TITLE
Allow layers to be disabled in PBS_ColorSplat

### ISSUE THREAD

**Issue by flarn2006** [Neos-Metaverse/NeosPublic#3203]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

Using the alpha channel for something other than opacity can be awkward. If you want to use less than four layers without having a fourth layer fill the material (possibly messing up metallic/smoothness) then the alpha channel is going to need to be one of them, unless you want to upload an image that's completely transparent. So what I'm suggesting, is to allow each layer to be toggled on or off, and when a layer is turned off, it will render as if that layer's channel in the color map is 0 everywhere.

Besides making some color maps less awkward to deal with, this will also enable layers to be turned on and off by LogiX.

### Describe alternatives you've considered

One alternative is to use ColorMask instead of ColorSplat, which doesn't have this problem; this, however, has the significant limitation where layers can only differ in color (and possibly texture).

**Comment by Frooxius** [comment]
I don't think this is the right approach for this, as it would most likely be wasteful - it will increase number of variants of the shader unnecessarily - the texture sampling already gives you RGBA data - if you don't want to use a particular channel, just don't have it in the texture. 

Better approach would be to visualize the texture differently, or perhaps switch how A channel is interpreted - inverting it instead, so transparent means that it's active and opaque that it's not.
